version.gradle.kts

```diff
@@ -24,19 +24,14 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-final def versions = [
-        errorProne      : "2.4.0",
-        errorProneJavac : "9+181-r4173-1", // taken from here: https://github.com/tbroyer/gradle-errorprone-plugin/blob/v0.8/build.gradle.kts
-        guava           : "31.0.1-jre",
-        checkerFramework: "3.21.0",
-        grpc            : "1.28.1",
-        pmd             : "6.41.0",
+val guavaVersion: String by extra("31.0.1-jre")
+val grpcVersion: String by extra("1.28.1")
 
-        junit5          : "5.8.2"
-]
+val errorProneCoreVersion: String by extra("2.4.0")
+val errorProneJavacVersion: String by extra("9+181-r4173-1")
+val checkerFrameworkVersion: String by extra("3.21.0")
 
-ext {
-    deps = [
-            "versions": versions
-    ]
-}
+val pmdVersion: String by extra("6.41.0")
+
+val apiGuardianVersion: String by extra("1.1.0")
+val junit5Version: String by extra("5.8.2")
```

Comment:
Please notice this red symbol at the end of the line. It's a broken EOF.

Please address here and in the rest of the files of this changeset.